MapboxDirections/MBRouteLeg.swift

```diff
@@ -173,6 +182,15 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     public let segmentCongestionLevels: [CongestionLevel]?
 
+    /**
+     An array containing the maximum speed along each road segment in the route leg geometry.
+
+     Speed limit data is available in [a number of countries and territories worldwide](https://docs.mapbox.com/help/how-mapbox-works/directions/).
```

Comment:
It might be worth mentioning that the returned speed limits may be [advisory speed limits](https://en.wikipedia.org/wiki/Advisory_speed_limit) on entrance and exit ramps where legal speed limits aren’t posted.